database/migrations/2018_07_27_000000_create_modules_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateModulesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('modules', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('collection', 32)->index();
```

Comment:
The collection field is set to 32 characters but the validation doesn't enforce this.
I presume the collection field would be filled from a multiselect so maybe this doesn't matter that much?